<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-UCD955FI" failures="0" tests="3" name="com.aspire.framework.pagefactory.TestNG.Baseclass.utilityclass.loginTest" time="0.002" errors="1" timestamp="2022-08-12T19:25:56 IST" skipped="2">
  <testcase classname="com.aspire.framework.pagefactory.TestNG.Baseclass.utilityclass.loginTest" name="verificationofinitias" time="0.000">
    <skipped/>
  </testcase> <!-- verificationofinitias -->
  <system-out/>
  <testcase classname="com.aspire.framework.pagefactory.TestNG.Baseclass.utilityclass.loginTest" name="logintoapp" time="0.002">
    <error message="&quot;G:\java program\eclipse examples\Framework\Data\data.xlsx&quot; (The filename, directory name, or volume label syntax is incorrect)" type="java.io.FileNotFoundException">
      <![CDATA[java.io.FileNotFoundException: "G:\java program\eclipse examples\Framework\Data\data.xlsx" (The filename, directory name, or volume label syntax is incorrect)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:216)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:111)
at com.aspire.framework.pagefactory.TestNG.Baseclass.utilityclass.UtilityClass.TestData(UtilityClass.java:24)
at com.aspire.framework.pagefactory.TestNG.Baseclass.utilityclass.loginTest.logintoapp(loginTest.java:41)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- logintoapp -->
  <system-out/>
  <testcase classname="com.aspire.framework.pagefactory.TestNG.Baseclass.utilityclass.loginTest" name="verificationoflogo" time="0.000">
    <skipped/>
  </testcase> <!-- verificationoflogo -->
  <system-out/>
</testsuite> <!-- com.aspire.framework.pagefactory.TestNG.Baseclass.utilityclass.loginTest -->
